crates/collateral-contract/CLI.md

```
@@ -0,0 +1,433 @@
+- Contract interaction errors
+
+## Configuration
+
+The CLI supports multiple network configurations:
+
+### Network Configurations
+
+- **Mainnet** (default):
+
+  - Chain ID: 964
+  - RPC URL: https://lite.chain.opentensor.ai:443
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)